artichoke-backend/src/extn/core/time/mruby.rs

```diff
@@ -141,7 +141,7 @@ unsafe extern "C" fn time_self_mkutc(mrb: *mut sys::mrb_state, _slf: sys::mrb_va
 unsafe extern "C" fn time_self_mktime(mrb: *mut sys::mrb_state, _slf: sys::mrb_value) -> sys::mrb_value {
     let args = mrb_get_args!(mrb, *args);
     unwrap_interpreter!(mrb, to => guard);
-    let args = args.iter().copied().map(Value::from);
+    let args = &args.iter().copied().map(Value::from).collect::<Vec<Value>>();
```

Comment:
I think it would be a bit more clear (and not rely on rustc promotion) to take the borrow on the vec in the below line:

```rust
let args = args.iter().copied().map(Value::from).collect::<Vec<Value>>();
let result = trampoline::mktime(&mut guard, &args);
```